### PR TITLE
Updated quick-reference, create wallets in deploy_testnet_example

### PR DESCRIPTION
**Pull request by NinoLipartiia**
`deploy_testnet_example.sh` creates wallets and requests some tokens, also added this to quick-reference

### CODE REVIEW

File: deploy_testnet_example.sh [hunk 1]
```diff
@@ -11,10 +11,26 @@ docker run --rm -v "$(pwd)":/code \
...
 TXFLAG="--node https://rpc.uni.juno.deuslabs.fi:443 --chain-id uni-3 --gas-prices 0.025ujunox --gas auto --gas-adjustment 1.3 --broadcast-mode block"
 
-# Make sure OWNER and USER wallets have enough JUNOX
-OWNER="$1"
-AGENT="$2"
-USER="$3"
+# Create wallets and make sure they have some JUNOX
+OWNER=owner$RANDOM
+AGENT=agent$RANDOM
+USER=user$RANDOM
```
Comment (Buckram123) [inline deploy_testnet_example.sh:17]:
```suggestion
OWNER=croncat-owner-test
AGENT=croncat-agent-test
USER=croncat-user-test
```
and delete keys for those before 
```bash
set -e
```
imo it's better to not fill keyring with test keypairs, what you think? @NinoLipartiia

File: quick-reference.md [hunk 2]
```diff
@@ -25,7 +25,25 @@ USER=creator-of-the-tasks-address
...
 ```
 Note, that `OWNER`'s balance must be enough for storing a wasm file.
-`USER` will also spend some tokens creating tasks.
+`USER` will spend some tokens creating tasks, `AGENT` only pays gas fees.
+
+Alternatively you can create new wallets and request some JUNOX from the faucet:
+```bash
+OWNER=owner$RANDOM
+AGENT=agent$RANDOM
+USER=user$RANDOM
```
Comment (Buckram123) [inline quick-reference.md:34]: same here

File: quick-reference.md [hunk 3]
```diff
@@ -152,12 +171,12 @@ junod query wasm contract-state smart $CONTRACT "$GET_TASKS_BY_OWNER" $NODE
...
 To get a task by the hash:
 ```bash
-GET_TASK='{"get_task":{"task_hash":"4905bb310073e83af6cd9c4c19f9f5782db79e7f8b08b4035b664d8f39d31dd7"}}'
+GET_TASK='{"get_task":{"task_hash":"2cfc83749ca11d1ea8461cf919a3eee7f0e7fc5246ab0694add1b54473d46b03"}}'
 junod query wasm contract-state smart $CONTRACT "$GET_TASK" $NODE
 ```
 To get a hash of the task:
 ```bash
-GET_TASK_HASH='{"get_task_hash":{"task":{"owner_id":"juno1qgdwpzngq8wtrd0xamfpr0fse7egrefye6ekuh","interval":"Immediate","boundary":{"start":null,"end":null},"stop_on_fail":false,"total_deposit":[{"denom":"ujunox","amount":"1"}],"actions":[{"msg":{"staking":{"delegate":{"validator":"juno14vhcdsyf83ngsrrqc92kmw8q9xakqjm0ff2dpn","amount":{"denom":"ujunox","amount":"400000"}}}},"gas_limit":150000}],"rules":null}}}'
+GET_TASK_HASH='{"get_task_hash":{"task":{"owner_id":"'$(junod keys show $OWNER -a)'","interval":"Immediate","boundary":{"start":null,"end":null},"stop_on_fail":false,"total_deposit":[{"denom":"ujunox","amount":"500000"}],"actions":[{"msg":{"staking":{"delegate":{"validator":"juno14vhcdsyf83ngsrrqc92kmw8q9xakqjm0ff2dpn","amount":{"denom":"ujunox","amount":"300000"}}}},"gas_limit":150000}],"rules":null}}}'
```
Comment (Buckram123) [inline quick-reference.md:179]: perfect

File: quick-reference.md [hunk 4]
```diff
@@ -152,12 +171,12 @@ junod query wasm contract-state smart $CONTRACT "$GET_TASKS_BY_OWNER" $NODE
 ```
 To get a task by the hash:
 ```bash
-GET_TASK='{"get_task":{"task_hash":"4905bb310073e83af6cd9c4c19f9f5782db79e7f8b08b4035b664d8f39d31dd7"}}'
+GET_TASK='{"get_task":{"task_hash":"2cfc83749ca11d1ea8461cf919a3eee7f0e7fc5246ab0694add1b54473d46b03"}}'
```
Comment (Buckram123) [inline quick-reference.md:174]: task hash will be different because `owner_id` will be different

**Review by Buckram123 — COMMENTED** [review]
Great job! Thank you for giving many examples of using this contract from cli. :)

File: quick-reference.md (hunk 4 above)
Comment (deveusss) [reply quick-reference.md:174]: awesome one)

**Comment by deveusss** [comment]
Good stuff, thank @NinoLipartiia :)

**Pull request merged**